plugins/woocommerce-admin/client/products/tour/product-tour.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { TourKit, TourKitTypes } from '@woocommerce/components';
+
+type ProductTourProps = {
+	onClose: () => void;
+};
+
+export const ProductTour: React.FC< ProductTourProps > = ( { onClose } ) => {
+	const tourConfig: TourKitTypes.WooConfig = {
+		placement: 'auto',
+		options: {
+			effects: {
+				spotlight: {
+					interactivity: {
+						enabled: false,
+					},
+				},
+				liveResize: {
+					mutation: true,
+					resize: true,
+				},
+			},
+		},
+		steps: [
+			{
+				referenceElements: {
+					desktop: `.woocommerce-product-form-tab__general .woocommerce-form-section__content`,
+				},
+				meta: {
+					name: 'story',
+					heading: __(
+						'📣 Tell a story about your product',
+						'woocommerce'
+					),
+					descriptions: {
+						desktop: __(
+							'The product form will help you describe your product field by field—from basic details like name and description to attributes the customers can use to find it on your store.',
+							'woocommerce'
+						),
+					},
+				},
+			},
+			{
+				referenceElements: {
+					desktop: `#tab-panel-0-pricing`,
+				},
+				meta: {
+					name: 'tabs',
+					heading: __( '✍️ Set up pricing & more', 'woocommerce' ),
+					descriptions: {
+						desktop: __(
+							'When done, use the tabs to switch between other details and settings. In the future, you’ll also find here extensions and plugins.',
```

Comment:
Does the language with 
> In the future, you'll also **find here extensions and plugins**.

Feel weird to anyone else? Not sure if it's just me.

Reply:
Definitely- good catch!  Updated to `In the future, you’ll also find extensions and plugins here.`. Sound okay to you @pmcpinto ?